engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3474,13 +3475,25 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
     public void execute(String[] args, ConsoleReader reader) throws Exception
     {
       try (AppPackage ap = newAppPackageInstance(new File(expandFileName(args[1], true)))) {
+        //Ignore all attribute descriptions since attributes are platform dependent and not app-package dependent.
+        for (Map.Entry<String, PropertyInfo> entry : ap.getDefaultProperties().entrySet()) {
+          if (entry.getKey().contains(".attr.")) {
```

Comment:
While creating the PropertyInfo don't load the Description for attributes. So you don't need both the for loops.